### DISCUSSION TITLE
Request to review code based on my learning on your YT channel

### DISCUSSION THREAD

**Discussion by gurunandan-bhat** [MarioCarrion/videos#13] [Show and tell]
Thanks to your explanation, I overcame my fear of errgroup and am now actively using it in production. Based on what I learnt on your channel, I wrote a small program using errgroup and I would sincerely appreciate your comments and review. Here is the gist: 
https://gist.github.com/Bhat-Gurunandan/aa52853f7e794613c5c11050c90191b0

Thanks once again for your work on spreading your expertise.

**Comment by MarioCarrion** [comment]
Hello!

Yes, what you implemented makes sense, really the only comment I have is the missing:
 
> 	f, err := os.Open(file)

which to be fair I didn't do that during the example, so by bad :smile:.

Thanks for the support! Take care.
